### PR TITLE
Fix linter for files inisde the dali_tf_plugin directory

### PR DESCRIPTION
**Pull request by JanuszL**
- fixes linter for files inisde the dali_tf_plugin directory
- adds the dali_tf_plugin directory to the python linter scan

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- fixes linter for files inisde the dali_tf_plugin directory
- adds the dali_tf_plugin directory to the python linter scan

### Affected modules and functionalities:
- dali_tf_plugin
- cmake/lint.cmake

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: dali_tf_plugin/dali_compile_flags.py [hunk 1]
```diff
@@ -14,22 +15,22 @@ def get_module_path(module_name):
...
     dali_include_flags = ''
     try:
         import nvidia.dali.sysconfig as dali_sc
-        dali_lib_path = dali_sc.get_lib_dir()
-        dali_include_flags=" ".join(dali_sc.get_include_flags())
-        dali_cflags=" ".join(dali_sc.get_compile_flags())
-        dali_lflags=" ".join(dali_sc.get_link_flags())
+        dali_include_flags = " ".join(dali_sc.get_include_flags())
+        dali_cflags = " ".join(dali_sc.get_compile_flags())
+        dali_lflags = " ".join(dali_sc.get_link_flags())
```
Comment (awolant) [inline dali_tf_plugin/dali_compile_flags.py:27]: Was `dali_lib_path` removed on purpose?

Comment (JanuszL) [reply dali_tf_plugin/dali_compile_flags.py:27]: Yes, it is a local variable that is not used anywhere in this funciton.

**Review by awolant — APPROVED** [review]

**Review by banasraf — APPROVED** [review]

**Pull request merged**